test_rules_scala.sh

```diff
@@ -734,6 +734,8 @@ $runner bazel run test:MixJavaScalaLibBinary
 $runner bazel run test:MixJavaScalaSrcjarLibBinary
 $runner bazel run test:ScalaBinary
 $runner bazel run test:ScalaLibBinary
+$runner bazel build test:ScalaBinary_statsfile
```

Comment:
Are you sure we need these? I might be mistaken but I think that `bazel build //test/...` will also catch it and we're doing it in the beginning

Reply:
Oh, good point!